vendor/vendor.json

```diff
@@ -615,10 +796,16 @@
 			"revisionTime": "2017-03-29T01:43:45Z"
 		},
 		{
-			"checksumSHA1": "42KMWr1GM3G/gC/3HjTt4XqB714=",
+			"checksumSHA1": "Y+HGqEkYM15ir+J93MEaHdyFy0c=",
+			"path": "golang.org/x/net/context",
+			"revision": "ffcf1bedda3b04ebb15a168a59800a73d6dc0f4d",
+			"revisionTime": "2017-03-29T01:43:45Z"
+		},
+		{
+			"checksumSHA1": "WHc3uByvGaMcnSoI21fhzYgbOgg=",
 			"path": "golang.org/x/net/context/ctxhttp",
-			"revision": "e90d6d0afc4c315a0d87a568ae68577cc15149a0",
-			"revisionTime": "2016-07-15T16:59:30Z"
+			"revision": "ffcf1bedda3b04ebb15a168a59800a73d6dc0f4d",
```

Comment:
As this was already used, one option to make the PR smaller would be to keep it to the same version. But seeing the version is quite old, probably a good time to update.